frontend/src/metabase/browse/models/BrowseModels.tsx

```diff
@@ -75,17 +87,23 @@ export const BrowseModels = () => {
         <BrowseSection>
           <Stack mb="lg" spacing="md" w="100%">
             {isEmpty ? (
-              <CenteredEmptyState
-                title={<Box mb=".5rem">{t`No models here yet`}</Box>}
-                message={
-                  <Box maw="24rem">{t`Models help curate data to make it easier to find answers to questions all in one place.`}</Box>
-                }
-                illustrationElement={
-                  <Box mb=".5rem">
-                    <img src={NoResults} />
-                  </Box>
-                }
-              />
+              <Stack spacing="lg" align="center" data-testid="empty-state">
+                <Box maw="45rem" w="100%">
+                  <ModelsVideo autoplay={0} />
+                </Box>
+                <Stack spacing="xs" maw="30rem">
+                  <Title
+                    order={2}
+                    align="center"
+                  >{t`Create models to clean up and combine tables to make your data easier to explore`}</Title>
+                  <Text align="center">{t`Models are somewhat like virtual tables: do all your joins and custom columns once, save it as a model, then query it like a table.`}</Text>
```

Comment:
what does 'it' refer to?

Reply:
🤷‍♂️ 

Probably "question", but I'd say that the more correct term would be "query". That's a nasty can of worms, imo.

Maybe we could remove this ambiguity altogether with: "Models are somewhat like virtual tables: do all your joins and custom columns once, save a model, then query it like a table."

But I'll leave that to the product team to figure out (with writers).